### PR TITLE
#37207 Plugin action store-uploaded-document

### PR DESCRIPTION
**Pull request by Klaas-Ritense**

### CODE REVIEW

File: zgw/documenten-api/src/main/kotlin/com/ritense/documentenapi/DocumentenApiPlugin.kt [hunk 1]
```diff
@@ -65,24 +67,70 @@ class DocumentenApiPlugin(
...
             inhoud = contentAsInputStream,
             beschrijving = description,
+            ontvangstdatum = getLocalDateFromMetaData(metadata, "receiptDate"),
+            verzenddatum = getLocalDateFromMetaData(metadata, "sendDate"),
             informatieobjecttype = informatieobjecttype,
         )
+        storeDocument(execution, request, storedDocumentUrl)
+    }
+
+    @PluginAction(
```
Comment (Klaas-Ritense) [inline zgw/documenten-api/src/main/kotlin/com/ritense/documentenapi/DocumentenApiPlugin.kt:95]: This new PluginAction is very similar to the one above. Initially I wanted to combine them. But then all the PluginActionProperty of the plugin above should become optional. Im not sure if thats what we want

File: zgw/documenten-api/src/test/kotlin/com/ritense/documentenapi/DocumentenApiPluginTest.kt [hunk 2]
```diff
@@ -105,4 +105,66 @@ internal class DocumentenApiPluginTest {
         assertEquals(LocalDateTime.of(2020, 1, 1, 1, 1, 1), emittedEvent.beginRegistratie)
     }
 
+    @Test
+    fun `should call cslient to store file`() {
```
Comment (Rick-Ritense) [inline zgw/documenten-api/src/test/kotlin/com/ritense/documentenapi/DocumentenApiPluginTest.kt:109]: typo

Comment (Klaas-Ritense) [reply zgw/documenten-api/src/test/kotlin/com/ritense/documentenapi/DocumentenApiPluginTest.kt]: fixed

File: zgw/documenten-api/src/main/kotlin/com/ritense/documentenapi/DocumentenApiPlugin.kt (hunk 1 above)
Comment (theo-ritense) [reply zgw/documenten-api/src/main/kotlin/com/ritense/documentenapi/DocumentenApiPlugin.kt:95]: I don't have a problem with having 2 separate actions, but the code duplication bothers me. You could make it more generic if you'd merge the properties of storeTemporaryDocument with the resource metadata into a new map and pass that to a private method that can be used by both actions.

**Review by theo-ritense — APPROVED** [review]

**Pull request merged**